src/components/options/providers/contextMenuItems.svelte

```diff
@@ -4,12 +4,13 @@ import Notiflix from "notiflix";
 import { createEventDispatcher } from "svelte";
 import { Sortable } from "sortablejs";
 
+import { isUrl } from "../../../js/shared/misc";
 import LocalStore from "../../../js/shared/local_store";
 import { StoreKey } from "../../../js/shared/constants";
 
 const dispatch = createEventDispatcher();
 
-// Auxiliary variable to store initial providers.
+// Auxiliary variable to store initial providers
```

Comment:
I don't know why you hate _periods_, but since [comments should be sentences](https://nedbatchelder.com/blog/201401/comments_should_be_sentences.html#:~:text=(Throughout%2C%20%E2%80%9Ccomment%E2%80%9D%20includes,end%20them%20with%20a%20period.), they must end with a period.